pkg/api/pod/warnings.go

```diff
@@ -357,3 +367,81 @@ func warningsForWeightedPodAffinityTerms(terms []api.WeightedPodAffinityTerm, fi
 	}
 	return warnings
 }
+
+func warningsForOverlappingVirtualPaths(podSpec *api.PodSpec) []string {
```

Comment:
might comment here that some of these are covered in validation, but some were missed